modules/light-clients/06-solomachine/client_state.go

```diff
@@ -231,10 +231,6 @@ func produceVerificationArgs(
 		return nil, nil, 0, 0, err
 	}
 
-	if cs.ConsensusState == nil {
-		return nil, nil, 0, 0, sdkerrors.Wrap(clienttypes.ErrInvalidConsensus, "consensus state cannot be empty")
```

Comment:
should we panic here or will we just let it panic when we try to use it later on?

Reply:
it will panic when we try to use `.ConsensusState`. I think it is okay to let that happen imo. The panic message gets redacted anyways